src/database/migrations/20241127141243-create-org-configs-table.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		await queryInterface.createTable('organization_configs', {
```

Comment:
@adithyadinesh0412 can we add created_by and updated _by. If the org have multiple admins they we need to track who is added or updated it. Can u check with Akash once

Reply:
We don't have those fields in org_extension @priyanka-TL . So I didn't include it here as well.